scripts/withdraw.js

```diff
@@ -5,18 +5,34 @@ const prepareWithdraw = require("./wrapper/withdraw")(web3, artifacts)
 const argv = require("./utils/default_yargs")
   .option("masterSafe", {
     type: "string",
-    describe: "Address of Gnosis Safe owning bracketSafes",
+    describe: "address of Gnosis Safe owning bracketSafes",
     demandOption: true,
   })
   .option("withdrawalFile", {
     type: "string",
     describe: "file name (and path) to the list of withdrawals",
-    demandOption: true,
   })
-  .option("allTokens", {
-    type: "boolean",
-    default: false,
-    describe: "ignore amounts from withdrawalFile and try to withdraw the maximum amount available for each bracket",
+  .option("from", {
```

Comment:
I would consider calling this option "brackets", as we used this syntax for the deposits and the complete-liquidity-provision script